cmd/thanos/query_frontend.go

```diff
@@ -43,7 +43,8 @@ type queryFrontendConfig struct {
 
 func registerQueryFrontend(app *extkingpin.App) {
 	comp := component.QueryFrontend
-	cmd := app.Command(comp.String(), "query frontend")
+	cmd := app.Command(comp.String(), "query frontend command implements a service deployed in front of queriers to improve query parallelisation and caching"+
```

Comment:
Should be `parallelization` IMO?

Reply:
It is not necessary to add `Query Frontend is fully stateless and horizontally scalable.` to the description. First sentence is fine.

Reply:
https://en.wiktionary.org/wiki/parallelisation bot are valid but I will use the more common `parallelization`